### PR TITLE
v01 Solution

### PR DESCRIPTION
**Pull request by home2ego911**
- [DEMO LINK](https://home2ego911.github.io/layout_product-cards-en/)
- [TEST REPORT LINK](https://home2ego911.github.io/layout_product-cards-en/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,13 +1,54 @@
...
+    <div class="card" data-qa="card">
+      <div class="image-wrapper card__image-wrapper">
+        <img
+          class="product-image"
+          src="./images/imac.jpeg"
+          alt="Apple iMac"
+        >
+      </div>
+
+      <h1 class="product-title card__product-title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h1>
```
Comment (Bohdan-Maksymiuk) [inline src/index.html:22]: h1 should be 1 per all page
what will be if there will be several cards in catalogue ?

File: src/index.html [hunk 2]
```diff
@@ -1,13 +1,54 @@
...
+
+      <p class="product-code card__product-code">Product code: 195434</p>
+
+      <div class="rating-wrapper card__rating-wrapper">
```
Comment (Bohdan-Maksymiuk) [inline src/index.html:26]: i believe that we can get rid of one of this selectors

File: src/styles/blocks/product/card.css [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  margin-bottom: 16px;
+}
+
+.card__rating-wrapper {
+  margin-bottom: 24px;
+}
+
+.card__price-wrapper {
+  margin-bottom: 16px;
+}
```
Comment (Bohdan-Maksymiuk) [inline src/styles/blocks/product/card.css:28]: these ones are block elements - no need for creating additional classes for external styles

**Review by Bohdan-Maksymiuk — CHANGES_REQUESTED** [review]
Good job - fix comments

File: src/index.html [hunk 4]
```diff
@@ -1,13 +1,54 @@
...
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./styles/index.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+
+  <body class="content">
+    <div class="card" data-qa="card">
+      <div class="card__image-wrapper">
+        <img
+          class="product-image"
```
Comment (marina-tilniak) [inline src/index.html:16]:
```suggestion
          class="card__product-image"
```

File: src/index.html [hunk 5]
```diff
@@ -1,13 +1,54 @@
...
+      <div class="card__rating-wrapper">
+        <div class="product-stars">
+          <div class="star product-stars__star"></div>
+          <div class="star product-stars__star"></div>
+          <div class="star product-stars__star"></div>
+          <div class="star product-stars__star"></div>
+          <div class="star product-stars__star"></div>
+        </div>
+
+        <p class="product-reviews">Reviews: 5</p>
```
Comment (marina-tilniak) [inline src/index.html:35]:
```suggestion
        <p class="card__product-reviews">Reviews: 5</p>
```

File: src/index.html [hunk 6]
```diff
@@ -1,13 +1,54 @@
...
+      </div>
+
+      <div class="card__price-wrapper">
+        <p class="product-price">Price:</p>
+
+        <p class="product-quantity">$2,199</p>
```
Comment (marina-tilniak) [inline src/index.html:41]:
```suggestion
        <p class="card__product-price">Price:</p>

        <p class="card__product-quantity">$2,199</p>
```

**Review by marina-tilniak — CHANGES_REQUESTED** [review]
let's use BEM naming

**Pull request closed without merging**